ENIGMAsystem/SHELL/Universal_System/Extensions/Steamworks/gameclient/utils/gc_leaderboards_score_downloaded_cookies.cpp

```diff
@@ -50,8 +50,8 @@ void get_leaderboard_entries(LeaderboardEntry_t leaderboard_entries[], unsigned
     leaderboard_entries_buffer << '\"' << "rank" << '\"' << ':';
     leaderboard_entries_buffer << std::to_string(leaderboard_entries[i].m_nGlobalRank) << ',';
     leaderboard_entries_buffer << '\"' << "userID" << '\"' << ':';
-    leaderboard_entries_buffer << '\"' << std::to_string(leaderboard_entries[i].m_steamIDUser.ConvertToUint64())
-                               << '\"';
+    leaderboard_entries_buffer << "\"@i64@" << std::hex << leaderboard_entries[i].m_steamIDUser.ConvertToUint64()
```

Comment:
Also, don't forget to remove these lines:

```cpp
/*
  TODO: When using the concatenation operator '+' to concatenate strings instead of '<<', the string content blowed up
  because if something I don't understand.
*/
// TODO: The userID is not the same as the steamID.
```